src/functions/module_statistical.ts

```diff
@@ -323,7 +323,7 @@ export const AVERAGEA = {
   ],
   returns: ["NUMBER"],
   computeFormat: (value1: Arg) => {
-    return Array.isArray(value1?.format) ? value1.format[0][0] : value1?.format;
+    return Array.isArray(value1) ? value1[0][0]?.format : value1?.format;
```

Comment:
I'm quite sure we can have object with only values, so `value1[0][0]` can be `= {value : smth}` (so in that case the format won't be there)

Reply:
the ```?``` in ```value1[0][0]?.format``` is useful because  ```value1[0][0]``` could be empty when you provide range out of existing cells e.g. ```A10000:B200000```